e2eTests/README.md

```diff
@@ -41,80 +26,60 @@ of how this setup works.
+for port in 5432 5672 5433 80 5673 15672 25672; do lsof -ti:$port | xargs kill -9; done
 ```
-</details>
-
-<details>
-  <summary>I made changes in `services/` but when I run latest are not there in the build</summary>
 
-Try removing the stale images.
+##### Remove stale images
 
 ```bash
 docker rmi tsd-proxy:latest tsd-api-mock:latest mq-interceptor:latest --force
 ```
-</details>
 
-<details>
-  <summary>I have `docker-compose` but I can't find `docker compose` command?</summary>
-
-In some cases, particularly on older Ubuntu distributions, you might find that you have `docker-compose` (V2) installed but not the `docker compose` subcommand. To resolve this, first ensure you have Compose V2 and then, you can create a symbolic link in the `cli-plugins` directory by executing the following commands:
-
-```bash
-mkdir -p ~/.docker/cli-plugins
-ln -sfn /usr/local/bin/docker-compose ~/.docker/cli-plugins/docker-compose
-```
+##### Docker Compose issues
 
-For migration guidance to Docker Compose, visit [migrate to docker compose](https://docs.docker.com/compose/migrate/). Also, for further discussions and troubleshooting, refer to the GitHub issue at https://github.com/docker/compose/issues/8630.
-</details>
+First, verify that `docker compose` is correctly installed. Ensure you have Compose V2.
 
-<details>
-  <summary>Docker fails to build mq-interceptor (dial tcp: lookup proxy.golang.org: i/o timeout) </summary>
+For migration guidance to Docker Compose, visit [migrate to docker compose](https://docs.docker.com/compose/migrate/).
 
-Do a docker daemon restart.
+In some cases, particularly on older Ubuntu distributions, you might find that you have `docker-compose` (V2) installed but not the `docker compose` subcommand. To resolve this, you can create a symbolic link in the `cli-plugins` directory by executing the following commands:
 
 ```bash
-sudo systemctl restart docker
+mkdir -p ~/.docker/cli-plugins
+ln -sfn /usr/local/bin/docker-compose ~/.docker/cli-plugins/docker-compose
 ```
-</details>
-
-<details>
-  <summary>I made a mess. I want to start from scratch.</summary>
 
-First, clean the e2eTests directory and all the submodules.
+For further discussions and troubleshooting, refer to the GitHub issue at https://github.com/docker/compose/issues/8630.
 
-```bash
-sudo rm -rf e2eTests/tmp e2eTests/bin docker-compose.yml &&
-./gradlew clean
-```
+##### Maybe try cleaning everything?
 
-And then let's prune the docker system and restarts the docker daemon.
+This prunes the docker system and restarts the docker daemon.
 
 ```bash
 docker stop $(docker ps -aq) && \
 docker rm $(docker ps -aq) && \
 docker rmi tsd-proxy:latest tsd-api-mock:latest mq-interceptor:latest --force && \
-lsof -ti:5432 -ti:5672 -ti:5433 -ti:80 -ti:5673 -ti:15672 -ti:25672 | xargs kill -9 \
+for port in 5432 5672 5433 80 5673 15672 25672; do lsof -ti:$port | xargs kill -9; done && \
 echo 'y' | docker system prune &&
 sudo systemctl restart docker
```

Comment:
We can consider using `-15`, which sends a `SIGTERM` signal. So, it allows processes to gracefully shut down. Or else what we can do it use entirely different ports in our local setup.

- 5434 (instead of 5432)
- 5674 (instead of 5672)
- 5435 (instead of 5433)
- 81 (instead of 80)
- 5675 (instead of 5673)
- 15673 (instead of 15672)
- 25673 (instead of 25672)